src/com/palmergames/bukkit/towny/listeners/TownyPlayerListener.java

```diff
@@ -790,7 +792,7 @@ public void onOutlawEnterTown(PlayerEnterTownEvent event) throws NotRegisteredEx
 			if (TownySettings.doTownsGetWarnedOnOutlaw()) {
 				TownyMessaging.sendPrefixedTownMessage(town, Translation.of("msg_outlaw_town_notify", outlaw.getFormattedName()));
 			}
-			if (TownySettings.canOutlawsEnterTowns()) {
+			if (TownySettings.canOutlawsEnterTowns() || TownyUniverse.getInstance().getPermissionSource().isTownyAdmin(event.getPlayer())) {
```

Comment:
I would move the isAdmin() test up to be in front of line 788. If this is an admin we need to bail out earlier.

Reply:
I had it here intentionally so that admins/others with the perm to bypass (which I just added) will still be warned that they are outlawed in case they are also players, but... we likely don't want admins announce if the server is announcing outlaws, so I'm adding the generic admin check on line 791 as well.